### PR TITLE
Api definitions top part

### PR DESCRIPTION
**Pull request by ivanzusko**

### CODE REVIEW

File: src/modules/Layout/Title/Title.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@import 'index.scss';
+
+.j-title {
+	font-size: 25px;
```
Comment (Vxplus) [inline src/modules/Layout/Title/Title.scss]: The headers should have been initialized already with standard sizes for each one, same with it's line heights, imo is not a good idea to have a size specific for this h1

Comment (ivanzusko) [inline src/modules/Layout/Title/Title.scss]: Yeap, agree

Comment (ivanzusko) [inline src/modules/Layout/Title/Title.scss]: @Vxplus changed

File: src/variables.scss [hunk 2]
```diff
@@ -15,6 +15,9 @@ $hf-box-shadow-header: 0 1px 2px 0 rgba(0, 0, 0, 0.1);
 /* Fonts */
 $hf-font-family-sans-serif: 'roboto', sans-serif;
 $hf-font-size-base: 14px;
+$hf-font-size-title: $hf-font-size-base*2;
+
+$hf-line-height-title: 1.32;
```
Comment (Vxplus) [inline src/variables.scss]: and advice will be to calculate the line-height out of the font base like bootstrap.

**Pull request merged**